src/dbnode/storage/namespace.go

```diff
@@ -1700,9 +1703,11 @@ func (n *dbNamespace) aggregateTiles(
 
 	n.log.Info("finished large tiles aggregation for namespace",
 		zap.String("sourceNs", sourceNs.ID().String()),
-		zap.Int("shards", len(targetShards)),
+		zap.Time("blockStart", blockStart),
+		zap.Time("end", opts.End),
```

Comment:
nit: since `blockStart` is truncated to block size, should we do the same for `opts.End` above? It seems to get implicitly truncated by `sourceBlockStart := blockStart; sourceBlockStart.Before(opts.End); sourceBlockStart = sourceBlockStart.Add(sourceBlockSize)` anyway; may be clearer to truncate it first?

Reply:
👍